cadc-vos/src/main/java/ca/nrc/cadc/vos/client/ClientTransfer.java

```diff
@@ -143,10 +135,10 @@ private ClientTransfer() { }
     /**
      * @param jobURL UWS job URL for the transfer job
      * @param transfer a negotiated transfer
-     * @param monitor monitor the job until complete (true) or just start
+     * @param transListener monitor the job until complete (true) or just start
      *  it and return (false)
      */
-    ClientTransfer(URL jobURL, Transfer transfer, boolean schemaValidation)
+    ClientTransfer(URL jobURL, Transfer transfer, boolean transListener)
     {
```

Comment:
The schemaValidation parameter here has been lost.... seems like it was replaced with transListener which is not used.